flow/Platform.h

```diff
@@ -108,7 +109,11 @@
  */
 #ifndef _MSC_VER
 #if defined(__GNUG__)
-#define __assume(cond) do { if (!(cond)) __builtin_unreachable(); } while (0)
+#define __assume(cond)                                                                                                 \
```

Comment:
Why does the indentation before the `\` look so weird? Is this an effect of mixing tabs and spaces?

Reply:
It looks like what it's doing is putting the `\` on the very last character of the line (120) and using spaces as the whitespace character.